packages/paste-core/components/table/stories/index.stories.tsx

```diff
@@ -1006,3 +1007,74 @@ export const Truncation = (): React.ReactNode => {
     </Table>
   );
 };
+
+export const CustomTable = (): React.ReactNode => (
+  <CustomizationProvider
+    baseTheme="default"
+    elements={{
+      TABLE: {fontFamily: 'fontFamilyCode'},
+      TH_HEAD_LARGE: {fontSize: 'fontSize50'},
+      TH: {fontWeight: 'fontWeightBold'},
+      TH_BODY: {fontWeight: 'fontWeightMedium'},
+      TH_FOOTER: {textDecoration: 'underline'},
+      TR_DESTRUCTIVE: {color: 'colorTextLinkDestructive', fontWeight: 'fontWeightBold'},
+      TR_NEUTRAL: {color: 'colorTextNeutral', fontFamily: 'fontFamilyText'},
+      TR: {color: 'colorTextWeak'},
+      TD_FOOT_HIGHLIGHT: {
```

Comment:
I see what you did here, you mixed default element names with custom element names. Neat condensed story

Reply:
I was hoping folks would leave feedback on this. Thanks, I figured it's a good starting point for this customization story, since there are so many components that go into composing a table. Figured it's maybe more robust to combine them.